app/assets/stylesheets/common.scss

```diff
@@ -101,6 +101,10 @@ a {
   }
 }
 
+.text-muted a {
```

Comment:
So `text-muted` is a bootstrap thing but we still need a rule for it? Is that because bootstrap just fades it but we want to set the colour or something?

Also are the `deemphasize` rules above this still needed?

Reply:
In bootstrap `text-muted` there is no highlight colour for links, just an underline on hover. We use muted text with multiple links in various places, so I decided to keep the link colouring for now. 

The deemphasize rules are still needed for standard-form help text, but will be removed after forms are refactored.